src/parser.py

```diff
@@ -101,7 +103,8 @@ def get_medias(auth, user_id, include_retweets, image_size, since, since_id, unt
     }
     capi = api.favorites if likes else api.user_timeline
     pbar = tqdm(desc='Resolving', unit=' tweets')
-    for tweet in tweepy.Cursor(capi, id=user_id, include_rts=include_retweets, include_entities=True, tweet_mode='extended', since_id=since_id, max_id=until_id).items():
```

Comment:
# include_entities=True
This option is deprecated in current Tweepy releases, and produces an error message, so it is removed.
In addition, now using screen_name=user_id inside tweepy.Cursor because the original, id=user_id, is no longer supported (at least in my build).